chain/network/src/lib.rs

```diff
@@ -1,5 +1,8 @@
 #[macro_use]
 extern crate lazy_static;
+extern crate strum;
+#[macro_use]
+extern crate strum_macros;
```

Comment:
No need for `extern crate` since Rust 2018 edition; just `use` macros and derives as regular functions or structs.